Jenkinsfile

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
Can you add an explanation to the commit message what this Jenkinsfile is supposed to be used for?
And is the root directory the best place for this file? Because it's not used as CI for ORT, but to use ORT in CI to process other projects.

Reply:
> Can you add an explanation to the commit message what this Jenkinsfile is supposed to be used for?

Uhm, what more information than "Add an initial Jenkinsfile to run the ORT analyzer on a Git project" would you require?

> And is the root directory the best place for this file? Because it's not used as CI for ORT, but to use ORT in CI to process other projects.

I was also briefly thinking about this, but couldn't come up with a good directory name. Simply "ci" would also be too generic / not very telling. Do you have a suggestion?

Reply:
> Uhm, what more information than "Add an initial Jenkinsfile to run the ORT analyzer on a Git project" would you require?

Maybe that's enough for the commit message, I'm looking forward to the README entry.

> I was also briefly thinking about this, but couldn't come up with a good directory name. Simply "ci" would also be too generic / not very telling. Do you have a suggestion?

I think "ci" would be fine. I'm also fine with moving it later if we have a better idea.